.circleci/config.yml

```diff
@@ -534,6 +534,39 @@ jobs:
       - store_test_results:
           path: build/test-results
 
+  verify-revenuecatui-snapshot-tests:
```

Comment:
I'd name this `verify-revenuecatui-snapshots` or `revenuecatui-snapshot-tests`, but both words make it sound redundant :P